gradle/scripts/lib/java-shade.gradle

```diff
@@ -118,11 +107,11 @@ configure(relocatedProjects) {
 
             // Include all other shaded JARs so that ProGuard does not trim the classes and methods
             // that are used actually.
-            injars tasks.shadedTestJar.archivePath
+            injars tasks.shadedTestJar.archiveFile.get().asFile
             relocatedProjects.each {
                 if (it != project && !it.hasFlags('no_aggregation')) {
-                    injars it.tasks.shadedJar.archivePath
-                    injars it.tasks.shadedTestJar.archivePath
+                    injars it.tasks.shadedJar.aarchiveFile.get().asFile
```

Comment:
Typo?

Reply:
- https://github.com/line/gradle-scripts/pull/178

I'll update this after pr above is get merged